alibi/utils/wrappers.py

```diff
@@ -1,6 +1,37 @@
 import numpy as np
 
-from functools import singledispatch, update_wrapper
+from typing import Callable, Union
+
+blackbox_wrappers = {'pytorch': None, 'tensorflow': None}
+"""dict: A registry for wrappers that cast the input to a black-box function to `np.array` object and the output to a
+frame-work specific tensor.
+"""
+
+
+def blackbox_wrapper(framework: str = 'tensorflow') -> Callable:
+    """
+    A decorator that registers a wrapper for a black-box model. In the  future, the functionality may be extended so
+    that the wrapper can also be parametrized by the algorithm name, if necessary.
+
+    Parameters
+    ----------
+    framework: {'pytorch', 'tensorflow'}
```

Comment:
Similar to transformers package, objects using TensorFlow use "TF" prefix. In the same package, "pt" and "tf" are used to indicate the backend. Maybe we should stay consistent and use those abbreviations instead of "pytorch" and "tensorflow".

Reply:
I don't see a point to be consistent with a entrily diferent package. I don't thin that's a convention, it's how the chose to name it.